src/Blocks/BlockCrops.h

```diff
@@ -81,9 +43,51 @@ class cBlockCropsHandler :
 					break;
 				}
 			}
+			return;
 		}
+
+		// Fully grown, drop the produce:
```

Comment:
`sheep` is a bad example for an uncountable noun, since it is actually really weird that it is uncountable in English. `flour` would have been a better one, I guess.

If I compare `drop the produce` with `drop the flour`, I don't feel much difference (linguistically), you can drop anything, countable or not. And making it `drop the crop's produce` doesn't seem to improve on it, rather, it's like duplicating the word.

So now we have:
 - `drop the produce`
 - `drop the product`
 - `drop the crop's produce`

Which one will it be? Or yet another one?

Now this entire PR will get stuck on a trivial language matter... :)